Makefile

```diff
@@ -72,19 +72,17 @@ extensions: build/$(ISODATE)-$(BUILD_ID)-chrome-prod.zip
 
 build/%-chrome-stage.zip:
 	@rm -rf build/chrome $@
-	SENTRY_DSN=$(SENTRY_DSN_STAGE) hypothesis-buildext \
-		conf/production.ini \
-		chrome \
-		--base 'https://stage.hypothes.is' \
+	hypothesis-buildext chrome \
+		--service 'https://stage.hypothes.is' \
+		--sentry-dsn "$(SENTRY_DSN_STAGE)" \
```

Comment:
It's a tiny thing, I know, but the other arguments use single quotes, so these should as well.